templates/default/fulldoc/html/full_list.erb

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html>
+<html <%= "lang=\"#{html_lang}\"" unless html_lang.nil? %>>
```

Comment:
We hardcoded lang="en" in our docs, but I think it makes sense to have this as a setup method.